### PR TITLE
Adjust default cpu/memory for openshift-extended-test based on usage

### PR DESCRIPTION
**Pull request by liangxia**
Get below resource usage from execution of https://github.com/openshift/release/pull/28364
Screenshot from 2022-05-06 15-08-18
Screenshot from 2022-05-06 15-08-40

/cc @kuiwang02 @jianzhangbjz @jhou1

### CODE REVIEW

**Comment by kuiwang02** [comment]
could you please check more golang test to see the cpu/memory usage?
could find current tests in https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/?type=periodic.

or,
could you please tell me how to check the cpu/memory?

I want to see more samples to check if the updated value is reasonable.
current value is from origin repo golang step which the openshift-tests-private inherit from.

**Comment by liangxia** [comment]
The resource usage of golang tests in this PR,
Screenshot from 2022-05-06 16-50-27
Screenshot from 2022-05-06 16-50-53

**Comment by kuiwang02** [comment]
could we set cpu as 2?

File: ci-operator/step-registry/openshift-extended/test/openshift-extended-test-ref.yaml [hunk 1]
```diff
@@ -39,8 +39,8 @@ ref:
       the subsequence step is not executed per https://docs.ci.openshift.org/docs/architecture/step-registry/#workflow
   resources:
     requests:
-      cpu: "3"
-      memory: 600Mi
+      cpu: "1"
+      memory: 1Gi
```
Comment (liangxia) [inline ci-operator/step-registry/openshift-extended/test/openshift-extended-test-ref.yaml:43]: The change happens in the `requests` part, it matters, but not the key point, as the `limits` is the one that matters the larges resources you can use.
Even if you require more/ or less of the cpu/memory, the tests will not fail, as OpenShift support allocating resources dynamically.
The update here is to make the resource usage more efficient. E.g, as you can see from the screenshots, we use less than 1 cpu most of the time, default requesting to 3 is a little waste of the resources.

**Comment by liangxia** [comment]
> could we set cpu as 2?

We can. But not necessarily at the moment. As you can in the screenshot, previously we request memory as 600Mi, but we still can use more than that, as OpenShift supports dynamic resources allocate. We just need to make sure the default (`requests`) resources can cover most of the execution time is enough.

**Comment by kuiwang02** [comment]
Understood. we can update it. and will monitor it.

could you please tell me how you monitor it so that I can monitor it after we merge the PR?

**Comment by liangxia** [comment]
> could you please tell me how you monitor it so that I can monitor it after we merge the PR?

Check that on the OCP cluster which is hosting your test pod. It can only be done when the tests is ongoing, as the project holding the test pod is deleted after the tests finished.

**Comment by kuiwang02** [comment]
Thanks. I find it in build farm cluster.

**Pull request merged**